tools/mitre/mitredb.py

```diff
@@ -430,9 +433,11 @@ def parse_json_techniques(technique_json, phases_table):
     if technique_json.get(const.DESCRIPTION_t):
         row[const.DESCRIPTION_t] = technique_json[const.DESCRIPTION_t]
     if technique_json.get(const.CREATED_j):
-        row[const.CREATED_t] = datetime.strptime(technique_json[const.CREATED_j], const.TIME_FORMAT)
+        row[const.CREATED_t] = core_utils.get_utc_strptime(technique_json[const.CREATED_j], const.TIME_FORMAT).replace(
+            tzinfo=timezone.utc)
```

Comment:
You don't need to do `.replace(tzinfo=timezone.utc)` here, that's already included inside the `get_utc_strptime(...)` function. Let me know if I'm wrong. 

```suggestion
        row[const.CREATED_t] = core_utils.get_utc_strptime(technique_json[const.CREATED_j], const.TIME_FORMAT)
```